src/Http/Authentication.Abstractions/src/AuthenticationProperties.cs

```diff
@@ -122,10 +121,10 @@ public bool IsPersistent
         }
 
         /// <summary>
-        /// Set a string value in the <see cref="Items"/> collection.
+        /// Sets or remove a string value from the <see cref="Items"/> collection.
```

Comment:
nit: sets or removes?  Not the best at grammar but I think the plurality is supposed to be the same

Reply:
```suggestion
        /// Set or remove a string value from the <see cref="Items"/> collection.
```